app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,27 @@
+class CommentsController < ApplicationController
+  before_action :authenticate_user!
+
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    @comment = @commentable.comments.new(comment_params)
+    @comment.user = current_user
```

Comment:
You could `.merge(user:current_user)` in comment_params and avoid needing this line. Or do `current_user.comments.build(...`